src/screens/RequestSwap/SelectSwapOption.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { View, StyleSheet, TouchableOpacity } from 'react-native';
+import { Layout, Text, Icon, Button } from '@ui-kitten/components';
+
+interface SelectSwapOptionProps {
+    navigation;
+    route;
+}
+
+export const SelectSwapOption = (props: SelectSwapOptionProps) => {
+    const [selectedOption, setSelectedOption] = useState<number>(0);
+
+    const onSelectHandler = (option) => {
+        setSelectedOption(option);
+    };
+
+    const iconDimensions = 50;
+
+    const SwitchSpecifically = () => {
+        return (
+            <TouchableOpacity
+                style={
+                    selectedOption === 0
+                        ? { ...styles.options, ...styles.selected }
+                        : styles.options
+                }
+                onPress={() => onSelectHandler(0)}
```

Comment:
if there's nothing else in the function `onSelectHandler`, we could just make it 
`onPress={()=> setSelectedOption(0)}` ?

Reply:
change made.